cypress/page/personal-information.page.ts

```diff
@@ -11,20 +12,27 @@ class PersonalInformation {
     private hobbiesMusic: string;
     private currentAddress: string;
     private clickSubmit: string;
+    private modalFullName: string;
+    //private modalEmail: string;
+
+
 
     constructor(){
+        this.fillingFormPageURL = "https://demoqa.com/automation-practice-form"
+        this.name = "#firstName"
+        this.lastName = "#lastName"
+        this.emailInput = "#userEmail-wrapper"
+        this.gender = "#genterWrapper > div.col-md-9.col-sm-12 > div:nth-child(1) > label"
```

Comment:
`#genterWrapper input` should work. That will return the 3 options as a list, then in the `SelectGender` method you can use the following to select the gender:
`cy.get(this.gender).filter(`\``[value="${gender}"]`\``).check({force: true})`